velox/docs/functions/spark/misc.rst

```diff
@@ -2,6 +2,13 @@
 Miscellaneous Functions
 ====================================
 
+.. spark:function:: monotonically_increasing_id() -> bigint
+
+    Returns monotonically increasing 64-bit integers. The generated ID is guaranteed to be monotonically increasing and unique, but not consecutive.
```

Comment:
Nit: please break such long line into two or more for being friendly to readers.

Reply:
Updated, thanks.